lib/mainloop-control.c

```diff
@@ -257,6 +257,21 @@ process_credentials(ControlConnection *cc, GString *command, gpointer user_data)
   control_connection_send_reply(cc, result);
 }
 
+static void
+show_files_present_in_config(ControlConnection *cc, GString *command, gpointer user_data)
+{
+  MainLoop *main_loop = (MainLoop *) user_data;
+  GlobalConfig *config = main_loop_get_current_config(main_loop);
+  GString *result = g_string_new("");
+
+  for (GList *v = config->file_list; v; v = v->next)
+    {
+      g_string_append(result, (gchar *) v->data);
+      g_string_append_c(result, '\n');
```

Comment:
A bit paranoid, but I suppose you want to use the output for scripting purposes. Then `\n` may not be the best separator character. The safest would be is to go with nullcharacter.  The cleanest would be to parse arguments, and have a `-0` option that uses nullcharacter as separator, similarly to find or xargs.
(not a change request, just an idea)

Reply:
Yes, that is a good idea. I can use ```xargs -0``` command in debun script if I have '\0' as separator.